parse/ValueRefPythonParser.h

```diff
@@ -71,6 +73,15 @@ struct value_ref_wrapper {
     const std::shared_ptr<const ValueRef::ValueRef<T>> value_ref;
 };
 
+template<typename T>
+std::unique_ptr<ValueRef::ValueRef<T>> object_to_value(const boost::python::object& obj) {
```

Comment:
would maybe call it `pyobject_to_vref` or `to_vref` ?

Reply:
OK